cityguide/source/app/src/main/java/org/kaaproject/kaa/demo/cityguide/fragment/AreasFragment.java

```diff
@@ -39,11 +39,11 @@
  */
 public class AreasFragment extends BaseFragment {
 
-    private View mWaitView;
-    private ListView mAreasListView;
+    View mWaitView;
```

Comment:
Field were made package-level to avoid creating synthetic accessor methods (see https://realm.io/news/360andev-jake-wharton-java-hidden-costs-android/ ). But I agree that it may be unclear and will make them private.